spec/features/edit_a_pom_spec.rb

```diff
@@ -3,119 +3,144 @@
+      signin_spo_user
+    end
 
-    visit edit_prison_pom_path('LEI', nomis_staff_id)
-    expect(page).to have_css('h1', text: 'Edit profile')
+    it "setting unavailable shows selected on re-edit", vcr: { cassette_name: 'prison_api/edit_poms_unavailable_check' } do
+      visit edit_prison_pom_path('LEI', nomis_staff_id)
+      expect(page).to have_css('h1', text: 'Edit profile')
 
-    expect(page).to have_field('status-conditional-unavailable', checked: true)
-  end
+      choose('working_pattern-2')
+      choose('Active but unavailable for new cases')
+      click_on('Save')
 
-  it "validates a POM when missing data", vcr: { cassette_name: 'prison_api/edit_poms_missing_check' } do
-    visit edit_prison_pom_path('LEI', fulltime_pom_id)
-    expect(page).to have_css('h1', text: 'Edit profile')
+      visit edit_prison_pom_path('LEI', nomis_staff_id)
+      expect(page).to have_css('h1', text: 'Edit profile')
 
-    expect(page.find('#working_pattern-ft')).to be_checked
+      expect(page).to have_field('status-conditional-unavailable', checked: true)
+    end
 
-    # The only way to trigger (and therefore cover) the validation is for a full-time POM
-    # to be edited to part time but not choose a working pattern.
-    choose('part-time-conditional-1')
-    click_on('Save')
+    it "validates a POM when missing data", vcr: { cassette_name: 'prison_api/edit_poms_missing_check' } do
+      visit edit_prison_pom_path('LEI', fulltime_pom_id)
+      expect(page).to have_css('h1', text: 'Edit profile')
 
-    expect(page).to have_css('h1', text: 'Edit profile')
-    expect(page).to have_content('Select number of days worked')
-  end
+      expect(page.find('#working_pattern-ft')).to be_checked
 
-  describe "making an inactive POM active", :js, vcr: { cassette_name: 'prison_api/edit_poms_activate_pom_feature' } do
-    let(:prison) { Prison.find('LEI') }
-    let(:other_prison) { create(:prison) }
-    let(:moic_integration_tests_staff_id) { 485_758 }
+      # The only way to trigger (and therefore cover) the validation is for a full-time POM
+      # to be edited to part time but not choose a working pattern.
+      choose('part-time-conditional-1')
+      click_on('Save')
 
-    before do
-      # create 2 inactive POM details records (to make POM inactive) - there was a bug that found the first(and wrong) one
-      create(:pom_detail, :inactive, prison: other_prison, nomis_staff_id: moic_integration_tests_staff_id)
-      create(:pom_detail, :inactive, prison: prison, nomis_staff_id: moic_integration_tests_staff_id)
+      expect(page).to have_css('h1', text: 'Edit profile')
+      expect(page).to have_content('Select number of days worked')
     end
 
-    it 'makes the pom have a status of active' do
-      visit "/prisons/LEI/poms"
-      click_link "Inactive staff (1)"
-      click_link 'Moic Integration-Tests'
+    describe "making an inactive POM active", :js, vcr: { cassette_name: 'prison_api/edit_poms_activate_pom_feature' } do
```

Comment:
Does this need to be a `:js` test, or is it just a remnant from when you were working on the test?

Feature tests without `:js` run faster because they don't need to run in Firefox.